keras_contrib/optimizers/lr_multiplier.py

```diff
@@ -0,0 +1,66 @@
+from keras.optimizers import Optimizer
+from keras.utils import get_custom_objects
+
+
+class LearningRateMultiplier(Optimizer):
+    """Optimizer wrapper for per layer learning rate.
+
+    This wrapper is used to add per layer learning rates by
+    providing per layer factors which are multiplied with the
+    learning rate of the optimizer.
+
+    Note: This is a wrapper and does not implement any
+    optimization algorithm.
```

Comment:
What about two examples? 
* One where you specify manually the learning rate by using the strings as keys `{'conv_1/kernel':0.5, 'conv_1/bias':0.1}`
* One where you programmatically set the learning rates by iterating through the layers of the model (for big models this is useful). I suppose that it should be possible with a for loop and getting the `layer.name` as the key of the dictionary.